MAINTAINERS.yml

```
@@ -0,0 +1,1055 @@
+    files:
+        - drivers/display/
+        - dts/bindings/display/
+        - include/display.h
+        - include/display/
+        - include/drivers/display.h
+    labels:
+        - "area: Display"
+
+Documentation:
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - dbkinder
+    files:
+        - doc/
+    files-regex:
+        - \.rst$
+        - /doc/
+    labels:
+        - "area: Documentation"
+
+"Drivers: ADC":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - anangl
+    files:
+        - drivers/adc/
+        - include/drivers/adc.h
+    labels:
+        - "area: ADC"
+
+"Drivers: ADC stm32":
+    # Provisional entry, to be corrected
+    collaborators:
+        - cybertale
+    files:
+        - drivers/adc/adc_stm32.c
+        - dts/bindings/iio/adc/st*stm32-adc.yaml
+
+"Drivers: Audio":
+    # Provisional entry, to be corrected
+    # TODO: spreadsheet not matching CODEOWNERS
+    status: maintained
+    maintainers:
+        - Vudentz
+    files:
+        - drivers/audio/
+        - include/audio/
+
+"Drivers: CAN":
+    # TODO: Waiting confirmation from @karstenkoenig to be added as collaborator
+    status: maintained
+    maintainers:
+        - alexanderwachter
+    collaborators:
+        - henrikbrixandersen
+    files:
+        - drivers/can/
+        - dts/bindings/can/
+        - include/drivers/can.h
+        - samples/drivers/CAN/
+        - tests/drivers/can/
+    labels:
+        - "area: CAN"
+
+"Drivers: Clock control":
+    status: orphaned
+    files:
+        - drivers/clock_control/
+        - dts/bindings/clock/
+        - include/clock_control.h
+        - include/dt-bindings/clock/
+        - tests/drivers/clock_control/
+    labels:
+        - "area: Clock control"
+
+"Drivers: Clock control: Nordic nRFx":
+    collaborators:
+        - nordic-krch
+    files:
+        - drivers/clock_control/*nrf*
+
+"Drivers: Console":
+    # Provisional entry, to be corrected
+    # TODO: spreadsheet not properly matching CODEOWNERS
+    status: maintained
+    maintainers:
+        - pfalcon
+    files:
+        - drivers/console/
+    labels:
+        - "area: Console"
+
+"Drivers: Counter":
+    # Provisional entry, to be corrected
+    # TODO: spreadsheet not properly matching CODEOWNERS
+    status: maintained
+    maintainers:
+        - nordic-krch
+    files:
+        - drivers/counter/
+        - include/drivers/counter.h
+    labels:
+        - "area: Counter"
+
+"Drivers: Crypto":
+    # Provisional entry, to be corrected
+    status: orphaned
+    files:
+        - drivers/crypto/
+        - dts/bindings/crypto/
+        - include/crypto/
+        - samples/drivers/crypto/
+    labels:
+        - "area: Crypto / RNG"
+
+"Drivers: DMA":
+    # Provisional entry, to be corrected
+    status: orphaned
+    files:
+        - drivers/dma/
+        - include/dt-bindings/dma/
+    labels:
+        - "area: DMA"
+
```

```suggestion
"Drivers: DMA stm32":
    # Provisional entry, to be corrected
    collaborators:
        - cybertale
    files:
        - drivers/adc/dma_stm32.c
        - dts/bindings/dma/st,stm32-dma.yaml
        - include/dt-bindings/dma/stm32_dma.h
```
@cybertale , can you confirm ?